src/client.js

```diff
@@ -491,6 +492,9 @@ export class StreamChat {
 				case 'patch':
 					response = await this.axiosInstance.patch(url, data, requestConfig);
 					break;
+				case 'options':
+					response = await this.axiosInstance.options(url);
```

Comment:
```suggestion
					response = await this.axiosInstance.options(url, requestConfig);
```

The `api_key` request header should be included, so the preflight request doesn't fail